src/avram/migrator/runner.cr

```diff
@@ -48,33 +48,35 @@ class Avram::Migrator::Runner
   end
 
   def self.drop_db(quiet? : Bool = false)
-    run "dropdb #{cmd_args}"
+    DB.connect(credentials.connection_string) do |db|
+      db.exec "DROP DATABASE IF EXISTS #{db_name}"
+    end
```

Comment:
If you've already dropped it, we can just silently ignore it. There's no real action to be taken.